docs/source/apps/aces/aces_css.ipynb

```diff
@@ -0,0 +1,319 @@
+    "# Optional imports\n",
+    "import os  # Used if setting a token as an environment variable"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To interface Superstaq via Cirq, we must first instantiate a service provider in `cirq-superstaq` with `Service()`. We then supply a Superstaq API token (or key) by either providing the API token as an argument of `css.Service()` or by setting it as an environment variable (see more details [here](https://superstaq.readthedocs.io/en/latest/get_started/basics/basics_css.html#Set-up-access-to-Superstaq%E2%80%99s-API))."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get cirq-superstaq service for Superstaq backend\n",
+    "service = css.Service()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## ACES"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To submit an ACES job, you will need the following information:\n",
+    "- target: device that you want to characterize.\n",
+    "- qubits: indices of the qubits to characterize.\n",
+    "- shots: number of shots to use per circuit to run.\n",
+    "- num_circuits: number of random circuits to sample in order to get the gate eigenvalues.\n",
+    "- mirror_depth: for each circuit, how many mirrored moments to include.\n",
+    "- extra_depth: for each circuit, how many random moments to include.\n",
+    "- method: execution method. If this is `\"noise-sim\"`, then optional arguments `noise` and `error_prob` must be passed.\n",
+    "\n",
+    "With this information, you can submit an ACES job through the `submit_aces` method. This will take care of constructing the circuits needed to perform the protocol and will execute them. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job_id = service.submit_aces(\n",
+    "    target=\"ss_unconstrained_simulator\",\n",
+    "    qubits=[0, 1],\n",
+    "    shots=100,\n",
+    "    num_circuits=5,\n",
+    "    mirror_depth=4,\n",
+    "    extra_depth=7,\n",
+    "    method=\"dry-run\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'c4428c69-5048-43f9-99e2-f005620d4134'"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "job_id"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since the circuits created by this protocol can take long to execute, specially if submitted to a real device, it might be convenient to save the job id somewhere. Once the jobs have finished running, calling `process_aces` and passing it the job id will compute the individual gate eigenvalues."
```

Comment:
```suggestion
    "We note that the circuits created by this protocol can take long to execute, especially if submitted to a real device, so it might be convenient to save the job id somewhere. Once the jobs have finished running, calling `process_aces` and passing it the job id will compute the individual gate eigenvalues."
```